src/test/java/com/faforever/client/replay/ReplayServiceTest.java

```diff
@@ -122,13 +139,17 @@
   private ModService modService;
   @Mock
   private MapService mapService;
+  @Mock
+  private EventBus eventBus;
+  @Mock
+  private ExecutorService executorService;
 
   @Before
   public void setUp() throws Exception {
     MockitoAnnotations.initMocks(this);
 
     instance = new ReplayService(new ClientProperties(), preferencesService, replayFileReader, notificationService, gameService,
```

Comment:
I would like to rename that `instance`... For example to that `replayService`

Reply:
Same to below, but it's just nice to have :-)

Reply:
This one is actually not from me. :) In all tests on this project, the respective object is stored in a property named `instance`.

Reply:
Yes, i know.